cmd/contacts.go

```diff
@@ -78,35 +76,3 @@ func (x *getContactsCmd) Execute(args []string) error {
 	output(res)
 	return nil
 }
-
-type addContactsCmd struct {
```

Comment:
Pulling this out because it's just too complicated to manually add a Contact via the CLI. In the future, you'll be able to "search for and add"... but this by itself doesn't seem useful.

Reply:
👍